modules/hub/src/ChannelsService.ts

```diff
@@ -1,56 +1,54 @@
+import { StateGenerator, types, Utils, Validator, big } from './Connext';
 import { redisCache } from './RedisClient'
-import { hasPendingOps } from './vendor/connext/hasPendingOps'
 import log from './util/log'
 import ChannelsDao from './dao/ChannelsDao'
 import Config from './Config'
 import ThreadsDao from './dao/ThreadsDao'
-import { BigNumber } from 'bignumber.js'
-import {
-  channelStateUpdateRowBigNumToString,
-  channelRowBigNumToString,
-  ChannelRow,
-  ChannelStateUpdateRowBigNum,
-} from './domain/Channel'
-import { Validator } from './vendor/connext/validator'
 import ExchangeRateDao from './dao/ExchangeRateDao'
-import { Big, toWeiBigNum } from './util/bigNumber'
-import { ThreadStateUpdateRow } from './domain/Thread'
 import { RedisClient } from './RedisClient'
 import { ChannelManager } from './ChannelManager'
-import {
-  ChannelStateUpdate,
-  UnsignedChannelState,
-  DepositArgs,
-  WithdrawalArgs,
-  ExchangeArgs,
-  convertChannelState,
-  convertThreadState,
-  ThreadState,
-  convertPayment,
-  PaymentArgs,
-  UpdateRequestBigNumber,
-  UpdateRequest,
-  ChannelStateBigNumber,
-  SyncResult,
-  WithdrawalParametersBigNumber,
-  ChannelStateBN,
-  InvalidationArgs,
-  Sync,
-  convertWithdrawalParameters,
-  convertWithdrawal,
-  convertArgs,
-} from './vendor/connext/types'
 import { prettySafeJson, Omit, maybe } from './util'
 import { OnchainTransactionService } from './OnchainTransactionService';
 import DBEngine from './DBEngine';
 import ThreadsService from './ThreadsService';
 import { SignerService } from './SignerService';
 import { OnchainTransactionRow } from './domain/OnchainTransaction';
 import ChannelDisputesDao from './dao/ChannelDisputesDao';
-import { StateGenerator } from './vendor/connext/StateGenerator';
 import { CoinPaymentsDao } from './coinpayments/CoinPaymentsDao'
 import { OnchainTransactionsDao } from './dao/OnchainTransactionsDao';
-
+import { BigNumber as BN } from 'ethers/utils'
+import { assetToWei } from 'connext/dist/lib/bn';
+import { ethers } from 'connext/node_modules/ethers';
```

Comment:
Why aren't we just importing `from 'ethers'`? This breaks the build on my computer + on CI